docs/compatibility.md

```diff
@@ -352,4 +352,10 @@ Casting from string to timestamp currently has the following limitations.
 - <a name="Footnote1"></a>[1] The timestamp portion must be complete in terms of hours, minutes, seconds, and
  milliseconds, with 2 digits each for hours, minutes, and seconds, and 6 digits for milliseconds. 
  Only timezone 'Z' (UTC) is supported. Casting unsupported formats will result in null values. 
- 
+
+### Constant Folding
+
+ConstantFolding is an operator optimization rule in Catalyst that replaces expressions that can
+be statically evaluated with their equivalent literal values. The RAPIDS Accelerator doesn't
+support excluding the rule `org.apache.spark.sql.catalyst.optimizer.ConstantFolding` as we rely on 
+this rule internally and excluding it may cause the query to produce undesired results
```

Comment:
This is no longer true after the other changes in this file.  Instead this should mention parts of the query may not run on the GPU without constant folding optimization enabled.